README.md

```diff
@@ -1,14 +1,14 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Say Hello Microverse
+# `My portfolio mobile version`
 
-> One paragraph statement about the project.
 
-![screenshot](images/screen-16.png)
+![screenshot](images/my-portfolio-1.png)
 
 Additional description about the project and its features.
 ### This is a simple say hello web Site
- - I added the Hello microverse in the h1 tags placed in the index.html file.
+ - I added the Hmtl, csll and linters.
+ - I parsed the figma design to get the design data.
  - Then added test.md and node_modules, to the .gitignore file
 
```

Comment:
- Provide a README file descriptive. Add information on how to run it and install your project locally.